pkgs/by-name/ss/sshified/package.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "sshified";
+  version = "1.2.1";
+
+  src = fetchFromGitHub {
+    owner = "hoffie";
+    repo = pname;
+    rev = "v${version}";
```

Helps with branch/tag confusion.

```suggestion
    rev = "refs/tags/v${version}";
```